src/test/java/com/wikia/webdriver/testcases/mercurytests/HTMLTitleTests.java

```diff
@@ -22,91 +32,91 @@
-                  "/wiki/Categoría:Regiones",
-                  "Categoría:Regiones - WikiDex - Wikia"
-          },
-          {
-                  "starwars",
-                  "/wiki/Main_Page",
-                  "Wookieepedia - Wikia"
-          },
-          {
-                  "starwars",
-                  "/wiki/Droid_starfighter",
-                  "Droid starfighter - Wookieepedia - Wikia"
-          },
-          {
-                  "starwars",
-                  "/main/category/Future_films",
-                  "Future_films"
-          },
-          {
-                  "starwars",
-                  "/main/section/Films",
-                  "Films"
-          },
-          {
-                  "mediawiki119",
-                  "/d/f/203236/latest",
-                  ""
-          },
-          {
-                  "mediawiki119",
-                  "/d/p/2706859396041803285",
-                  ""
-          },
-          {
-                  "mediawiki119",
-                  "/d/u/27334045",
-                  ""
-          },
-          {
-                  "dnd4",
-                  "/wiki/Dungeons_&_Dragons",
-                  "Dungeons & Dragons - D&D4 Wiki - Wikia"
-          }
+      {
+          "sktest123",
+          "/wiki/Sktest123_Wiki",
+          "Sktest123 Wiki - Wikia"
+      },
+      {
+          "sktest123",
+          "/wiki/Style-5H2",
+          "Style-5H2 - Sktest123 Wiki - Wikia"
+      },
+      {
+          "sktest123",
+          "/wiki/TestDisplayTitle",
+          "testing abc - Sktest123 Wiki - Wikia"
+      },
+      {
+          "sktest123",
+          "/wiki/Category:Premium_Videos",
+          "Category:Premium Videos - Sktest123 Wiki - Wikia"
+      },
+      {
+          "sktest123",
+          "/wiki/Category:Non-premium_Videos",
+          "Category:Non-premium Videos - Sktest123 Wiki - Wikia"
+      },
+      {
+          "sktest123",
+          "/wiki/Category:Premium",
+          "Category:Premium - Sktest123 Wiki - Wikia"
+      },
+      {
+          "es.pokemon",
+          "/wiki/WikiDex",
+          "WikiDex - Wikia"
+      },
+      {
+          "es.pokemon",
+          "/wiki/Lista_de_Pokémon",
+          "Lista de Pokémon - WikiDex - Wikia"
+      },
+      {
+          "es.pokemon",
+          "/wiki/Categoría:Regiones",
+          "Categoría:Regiones - WikiDex - Wikia"
+      },
+      {
+          "starwars",
+          "/wiki/Main_Page",
+          "Wookieepedia - Wikia"
+      },
+      {
+          "starwars",
+          "/wiki/Droid_starfighter",
+          "Droid starfighter - Wookieepedia - Wikia"
+      },
+      {
+          "starwars",
+          "/main/category/Future_films",
+          "Future_films"
+      },
+      {
+          "starwars",
+          "/main/section/Films",
+          "Films"
+      },
+      {
+          "mediawiki119",
+          "/d/f/203236/latest",
+          "Discussions - Mediawiki 1.19 test Wiki - Wikia"
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Discussions - Mediawiki 1.19 test Wiki - Wikia" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)